checklist.md

```diff
@@ -121,12 +121,38 @@ This part of the checklist is not applicable if you use a [Tradecloud One Connec
 Before starting the actual implementation, you need to verify the capabilities and requirements of your ERP system for an integration.
 
 - [ ] **Use a _delivery schedule_ or _single delivery_ per order line?**  
-  A delivery schedule consists of one or more deliveries, having a position, date & quantity, for the same item, prices and terms within one order line.
-  If your ERP system does not natively support multiple deliveries per order line, you should use the so called _single delivery_ feature. Else, we advise to use delivery schedules.
+
+  **Delivery Schedule vs. Single Delivery:**
+  
+  - **Delivery Schedule**: Multiple deliveries per order line (default method)
+    - Best for ERP systems that natively support multiple deliveries per order line (e.g., SAP)
+    - Provides more flexibility for complex delivery arrangements
+    - Maintains all deliveries within a single order line
+  
+  - **Single Delivery**: One delivery per order line
+    - Best for ERP systems that only support one delivery date/quantity per order line
+    - Tradecloud will automatically split delivery schedules into separate order lines
```

Comment:
Tradecloud will automatically split delivery schedules into separate order lines in the communication with your ERP system.